domain/src/main/java/org/oppia/android/domain/auth/AuthenticationController.kt

```diff
@@ -36,12 +34,16 @@ class AuthenticationController @Inject constructor(
           )
         }
       } catch (e: TimeoutCancellationException) {
-        // Handle timeout exception
         deferredResult.complete(
-          AsyncResult.Failure(TimeoutException("Authentication attempt timed out"))
+          AsyncResult.Failure(IllegalStateException(e))
         )
       }
     }
     return deferredResult
   }
+
+  companion object {
+    /** The amount of time the authentication task should run before timing out. */
+    const val AUTHENTICATION_TIMEOUT_MILLIS = 30_000L
```

Comment:
```suggestion
    private const val AUTHENTICATION_TIMEOUT_MILLIS = 30_000L
```

Prefer minimizing access if it doesn't need to be public.

Reply:
Updated